### PR TITLE
Fixes the Nano S issues on Onboarding

### PR DESCRIPTION
**Pull request by valpinkman**
LL-1152

### CODE REVIEW

**Comment by gre** [comment]
maybe we need to set it to `2 / PixelRatio.get()` so it's exactly 2 pixels. because i think we're in point domain here.

**Comment by gre** [comment]
We also need a new graphics.

File: src/components/SelectDevice/index.js [hunk 1]
```diff
@@ -201,18 +214,23 @@ class SelectDevice extends Component<OwnProps, State> {
     );
 
     const hasUSBSection = Platform.OS === "android" || other.length > 0;
+    const isNanoSOnOnboarding = isNanoS && onboarding;
 
     return (
       <View>
-        {ble.length === 0 ? (
+        {isNanoS && onboarding ? (
```
Comment (gre) [inline src/components/SelectDevice/index.js]: I think the condition can just be `isNanoS` , you can also remove `isNanoSOnOnboarding`

File: src/screens/Onboarding/steps/pair-new.js [hunk 2]
```diff
@@ -28,16 +28,23 @@ class OnboardingStepPairNew extends Component<OnboardingStepProps> {
   pairNew = () => this.props.navigation.navigate("PairDevices");
 
   render() {
+    const { deviceModelId, t } = this.props;
+    const isNanoS = deviceModelId === "nanoS";
```
Comment (gre) [inline src/screens/Onboarding/steps/pair-new.js]: **/!\** it does not include the Blue and should!

( rename it to usbOnly )

**Comment by gre** [comment]
after Valentin fixes this, reminder that we need to test with a Blue, (the design won't match, the wording should tho)

**Comment by valpinkman** [comment]
@gre Left To Do

- Verify address message still shows wrong text
- Started to contextualize the DeviceJob with deviceModelId, but did not integrate the textes or the correct illustration

I think I've covered the other cases

**Comment by gre** [comment]
Edit Device name exists only for NanoX over BLE

**Comment by gre** [comment]
@Arnaud97234 ready for review

**Pull request merged**